Libplanet/Crypto/PrivateKey.cs

```diff
@@ -269,6 +265,29 @@ private static ECPrivateKeyParameters GenerateKeyParam()
             return gen.GenerateKeyPair().Private as ECPrivateKeyParameters;
         }
 
+        private static ECPrivateKeyParameters GenerateKeyFromBytes(byte[] privateKey)
+        {
+            var param = new ECPrivateKeyParameters(
+                "ECDSA",
+                new BigInteger(1, privateKey),
+                GetECParameters()
+            );
+
+            var key = new PrivateKey(param);
+            try
+            {
+                var publicKey = key.PublicKey;
+            }
+            catch (ArgumentException)
+            {
+                throw new InvalidOperationException(
```

Comment:
I believe `ArgumentException` would be more suitable here.

```suggestion
                throw new ArgumentException(
```